docs/FAQ.md

```diff
@@ -1,80 +1,102 @@
 
 **Is there any pre-requisite to be a part of Twindle?**
+
 - No, you can contribute to the project in these ways:
-   Team: Helping each other, speaking up, pair programming, making and sharing videos (for this project).
-   Github: Updating the first PR and README.md, posting questions, or discussing in the issues created.
+  Team: Helping each other, speaking up, pair programming, making and sharing videos (for this project).
+  Github: Updating the first PR and README.md, posting questions, or discussing in the issues created.
 
-### Google Meet 📹
+## Google Meet 📹
 
 **Rules**
+
 - Mute yourself
 - Unmute only if you are speaking
 - Stay in a noise-free environment
 - If you are in a noisy environment don't speak
 
 **What will be the Google Meet timing?**
+
 - 5 pm IST
 
 **How to know what happened during that Google meet if I missed?**
+
 - Someone will take notes and share it with the team.
 - You can also watch the videos on this [list](https://youtu.be/i90UeTiEKQk) of the YT channel.
 
-### Github :octocat:
+## Github :octocat:
 
 **What to do next after creating a GitHub account?**
+
 - Fork the project from this [link](https://github.com/twindle-co/twindle)
 - Create the first PR.
 
 **How to fork?**
-- We'll do a video about it, and we can share the link in this answer.
+
+- This is covered in the video for making first pull request.
 
 **How to create first PR?**
-- Pictures of step by step [here](https://github.com/twindle-co/twindle/issues/57)
-- Video of PR steps:  [![here](http://img.youtube.com/vi/bzaBiQQl6fU/0.jpg)](https://youtu.be/bzaBiQQl6fU)
+
+- Pictures of step by step: [click here](https://github.com/twindle-co/twindle/issues/57)
+- Video of PR steps: [click here](https://youtu.be/bzaBiQQl6fU)
+
+**How to create second PR?**
+
+- Video tutorial: [click here](https://www.youtube.com/watch?v=bPBTumdMhyQ&ab_channel=Twindle)
+- Picture of step by step: [click here](https://github.com/twindle-co/twindle/issues/163)
+
+**How to create third PR?**
+
+- Video tutorial: [click here](https://www.youtube.com/watch?v=U2bOwEY-vKo&ab_channel=Twindle)
 
 **What are the two necessary tasks for being a part of Twindle (through Github)?**
+
 - Create a new file `firstname_lastname.md` inside the `firstpr` directory, commit your change and raise your first Pull Request.
 - Make few changes in the `firstname_lastname.md` file, commit, and use a "CLI" to raise your second Pull Request.
 
-### Logo 🎨
+### Youtube 💻
+
+Find all tutorials and recorded meets on this [channel](https://www.youtube.com/channel/UCKxUmbHq5P5pd5IyUiZ8MHA)
+
+## Logo 🎨
 
 **Where to make the logo from?**
+
 - May use free platforms like Canva, Gimp, or Inkspace
 - Use photoshop (if you already have the software)
 
 **How to share the logo I made?**
-- Comment on the issue **[#4 "We need a logo"](https://github.com/twindle-co/twindle/issues/4)** raised by johnjacobkenny
+
+- Comment on the issue **#4 "We need a logo"** raised by johnjacobkenny
```

Comment:
You removed the link ?

Reply:
No, I made changes in wiki and copied from there. Someone else must've done this